HQSmokeTests/testPages/messagingPage.py

```diff
@@ -150,9 +153,22 @@ def send_broadcast_message(self):
                                                                              + fetch_random_string())
         self.wait_to_click(By.XPATH, self.send_broadcast)
         self.driver.refresh()
+        
+        select = Select(self.driver.find_element(By.XPATH, self.broadcast_select))
+        select.select_by_value('100')
+        
+        time.sleep(5)
```

Comment:
having sleeps is definitely an anti-pattern and selenium does have some mechanisms to work around this: https://www.selenium.dev/documentation/webdriver/waits/